db/bigtable_eth1.go

```diff
@@ -4048,6 +4455,18 @@ func (bigtable *Bigtable) GetMethodLabel(id []byte, invokesContract bool) string
 	return method
 }
 
+// get a method label for its byte signature with defaults
+func (bigtable *Bigtable) GetAddressLabel(id string, invoke_overwrite types.ContractInteractionType) string {
+	switch invoke_overwrite {
+	case types.CONTRACT_CREATION:
+		return "Contract Creation"
+	case types.CONTRACT_DESTRUCTION:
+		return "Contract Destruction"
```

Comment:
I changed the max column width to address this; As we're carefully truncating every output anyways and the number of affected tables is limited, I thinks that's justified instead of thinking about alternative wording.